app/javascript/react/screens/App/Overview/screens/PlanWizard/components/PlanWizardVMStep/helpers.js

```diff
@@ -21,7 +21,6 @@ const manageBlankReason = vm => {
 };
 
 const manageOddCSVImportErrors = (vm, vmIndex, uniqueIds) => {
-  manageDuplicateVMRows(vm, vmIndex, uniqueIds);
```

Comment:
If we remove this, duplicate invalid rows will show up in the table, is this desired?

<img width="730" alt="manageiq__migration" src="https://user-images.githubusercontent.com/15141412/45505897-d0ee8680-b75b-11e8-9639-fb72cbd6ecc9.png">

Reply:
The duplicate rows in general are never eliminated from the table.
`manageDuplicateVMRows` just adds the 'duplicate' status to the VM

In case of invalid rows, it really does not matter if we mark duplicate rows as duplicates or not, because they are invalid and unselectable anyway.

Reply:
Whoops, I take back my 👍 😉 

Looks like if a VM is in another plan, and I import a CSV with duplicate rows of that VM, I get the following

<img width="1920" alt="fullscreen_9_13_18__2_04_pm" src="https://user-images.githubusercontent.com/15141412/45506741-24fa6a80-b75e-11e8-954e-da35d2de0d93.png">

Reply:
Ok, good catch :)

We will need `manageDuplicateVMRows` with a few adjustments